src/EditorFeatures/CSharpTest/GoToNextAndPreviousMember/CSharpGoToNextAndPreviousMemberTests.cs

```diff
@@ -0,0 +1,571 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Editor.CommandHandlers;
+using Microsoft.CodeAnalysis.Editor.UnitTests.Workspaces;
+using Roslyn.Test.Utilities;
+using Roslyn.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests
+{
+    public class CSharpGoToNextAndPreviousMemberTests
+    {
+        [Fact, Trait(Traits.Feature, Traits.Features.GoToNextAndPreviousMember)]
+        [WorkItem(4311, "https://github.com/dotnet/roslyn/issues/4311")]
+        public void EmptyFile()
+        {
+            var code = @"$$";
+            Assert.Null(GetTargetPosition(code, next: true));
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.GoToNextAndPreviousMember)]
```

Comment:
Does this need WpfFact? I'm not sure if TestWorkspaceFactory implicitly has demands of that. (And yes, we'll fix that.)